.ruby-version

```diff
@@ -1 +1 @@
-2.0.0-p648
\ No newline at end of file
+2.5.1
```

Comment:
Would you be able to revert this change? I want to test on the earliest supported Ruby which is currently 2.0.0 although may update to 2.3.7 soon.

Reply:
Ah, sorry, missed your comment.